package.json

```diff
@@ -23,12 +23,16 @@
     "prepublishOnly": "yarn build",
     "test": "yarn jest",
     "test:watch": "yarn jest --watchAll",
-    "storybook": "start-storybook -p 6006"
+    "storybook": "start-storybook -p 6006",
+    "storybook-ssl": "start-storybook -p 6006 --https --ssl-cert ./server.cert --ssl-key ./server.key"
   },
   "repository": "cognitedata/gearbox.js",
   "homepage": "https://github.com/cognitedata/gearbox.js#readme",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cognite/react-auth": "^2.1.2",
```

Comment:
Dev-dependency? We should require the user of the AssetScanner to be already authenticated.

Reply:
I'll remove this, we don't need it at all